src/views/ManageAssets.vue

```diff
@@ -5,11 +5,25 @@
     </NavBar>
     <div class="manage-assets_search form wrapper">
       <div class="input-group">
-        <SearchIcon /><input type="text" autocomplete="off" class="form-control form-control-sm" v-model="search" placeholder="Search for an Asset" />
+        <SearchIcon /><input type="text" autocomplete="off" class="form-control form-control-sm" v-model="search" @keyup="sortAssets" placeholder="Search for an Asset" />
       </div>
       <router-link to="/settings/manage-assets/custom-token">Add Custom Token</router-link>
-    </div>
+      <div v-if="assets.length === 0" class="mt-3 d-flex">
+        <div class="manage-assets_noneTop">
+          <h4 class="manage-assets_noneText">Can't find this token</h4>
+          <h4 class="manage-assets_customText">Add Custom ERC20 tokens.</h4>
+          <h5>Learn how</h5>
+        </div>
+      </div>
     <div class="manage-assets_list">
+       <div v-if="assets.length === 0" class="mt-5 d-flex">
+        <div class="manage-assets_noneBottom">
+          <h5 class="manage-assets_noneBottomText">INQUIRE</h5>
+          <h4 class="manage-assets_bottomEnable">Enable other tokens</h4>
```

Comment:
Where are the links?
These shouldn't have custom classes by the way, it should just use `<a>` tag
a link should be a link. If design presents the link differently, it's being inconsistent

Reply:
Yeah these will be changed, need to talk to thessy about them